neo4j/_async/work/session.py

```diff
@@ -239,6 +242,106 @@ async def run(self, query, parameters=None, **kwargs):
 
         return self._auto_result
 
+    async def query(self, query, parameters=None, **kwargs):
+        """
+        :param query: cypher query
+        :type query: str, neo4j.Query
+        :param parameters: dictionary of parameters
+        :type parameters: dict
+        :param kwargs: additional keyword parameters
+        :returns: a new :class:`neo4j.QueryResult` object
+        :rtype: QueryResult
```

Comment:
I like to separate param docs, return docs, and raises docs each by a blank line for better readability.

```suggestion

        :returns: a new :class:`neo4j.QueryResult` object
        :rtype: QueryResult
```

Reply:
Applies to other places as well, won't comment.